tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_personalization_RTL.py

```diff
@@ -0,0 +1,150 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I think this failed due to memory pressure.  we're planning to run the RTL tests on 512 or 1024MB config to avoid these issues.